notebooks/Query_butler_WEP_outputs.ipynb

```diff
@@ -0,0 +1,321 @@
+{
```

Comment:
<p>You should specify that this particular notebook runs on the summit RSP.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/aos_analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>